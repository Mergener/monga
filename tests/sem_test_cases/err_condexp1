function foo(x : int): int {
    return (x > 6) ? 5 : 4.2;
}