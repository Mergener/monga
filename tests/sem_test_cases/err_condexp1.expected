[Error] Module tests/sem_test_cases/err_condexp1, line 2, column 30: Ambiguous result type for conditional expression. The 'then' type (preceded by the ?) must be assignable from a value of 'else' type (preceded by the ':').
Semantic errors were caught during analysis.
